auth/src/subscription.rs

```diff
@@ -0,0 +1,19 @@
+pub mod stripe_price_ids {
+    /// The price ID of the recurring AWS RDS instance product.
+    /// TODO: this is set to the id of the test product, set to real product before releasing.
+    pub const AWS_RDS_INSTANCE_RECURRING: &str = "price_1OIS06FrN7EDaGOjaV0GXD7P";
+}
+
+pub trait SubscriptionItemExt {
+    fn price_id(&self) -> String;
+}
+
+impl SubscriptionItemExt for shuttle_common::backends::subscription::SubscriptionItem {
+    fn price_id(&self) -> String {
+        match self {
+            shuttle_common::backends::subscription::SubscriptionItem::AwsRds => {
+                stripe_price_ids::AWS_RDS_INSTANCE_RECURRING.to_string()
+            }
+        }
+    }
+}
```

Comment:
This needs to be dynamic, checking whether we are on prod and returning the correct price_id. I have created a ticket for this and will implement it in a follow-up after the production price ID has been created.

Reply:
Refactored to taking in an arg for the price id, so this is no longer relevant.